lib/src/note/positioned_note.dart

```diff
@@ -56,16 +58,16 @@ final class PositionedNote extends Note {
   /// ```
   @override
   PositionedNote transposeBy(Interval interval) {
-    final transposedNote = super.transposeBy(interval);
+    final transposedNote = note.transposeBy(interval);
 
     return transposedNote.inOctave(
       octaveFromSemitones(
-        semitonesFromRootHeight +
+        semitones +
             interval.semitones -
-            // We don't want to take the accidental into account when
-            // calculating the octave height, as it depends on the note name.
-            // This correctly handles cases with the same number of accidentals
-            // but different octaves (e.g., C♭4 but B3).
+// We don't want to take the accidental into account when
+// calculating the octave height, as it depends on the note name.
+// This correctly handles cases with the same number of accidentals
+// but different octaves (e.g., C♭4 but B3).
```

Comment:
Do you prefer the comment untabbed?

Reply:
Must have been my editor or dart format!

Reply:
Then: 😄 

```suggestion
            // We don't want to take the accidental into account when
            // calculating the octave height, as it depends on the note name.
            // This correctly handles cases with the same number of accidentals
            // but different octaves (e.g., C♭4 but B3).
```